packages/mongo/package.js

```diff
@@ -100,7 +100,7 @@ Package.onTest(function (api) {
   api.addFiles('collection_tests.js', ['client', 'server']);
   api.addFiles('collection_async_tests.js', ['client', 'server']);
   api.addFiles('observe_changes_tests.js', ['client', 'server']);
-  api.addFiles('oplog_tests.js', 'server');
   api.addFiles('oplog_v2_converter_tests.js', 'server');
   api.addFiles('doc_fetcher_tests.js', 'server');
+  api.addFiles('oplog_tests.js', 'server');
```

Comment:
It should not be needed if the `oplogOptionsTest` would clear after itself.

Reply:
But it does not hurt either! ;)

Reply:
That's a matter of personal preference - I suggest not to touch anything not related to the PR to make it easier to review and then easier to compare (or bisect if needed).

Reply:
okay, I will move it again!

Reply:
Fixed!